src/v2/Utils/user.ts

```diff
@@ -37,6 +37,17 @@ export function userIsAdmin(user?: User): boolean {
   return isAdmin
 }
 
+export function userIsTeam(user?: User): boolean {
+  const isTeam = Boolean(user && user.type === "Team" ? true : false)
```

Comment:
Is it ever the case that `user.type === "Team"`? I think the [`type` property](https://github.com/artsy/gravity/blob/3d89832cca98bd267a06d9caf9f63cc1cffa8c50/app/models/domain/user.rb#L575) is just a legacy thing from when `User`s and `Admin`s were different models. What you might want to check here is `roles`.

Reply:
Yes this looks correct -- `roles` is not currently available on the user, diging into this further.

Reply:
Added this related PR to passport so we can access the `roles` field in Force: https://github.com/artsy/artsy-passport/pull/154